ignite/handlers/checkpoint.py

```diff
@@ -200,6 +200,8 @@ def score_function(engine):
 
     """
 
+    filename_pattern = "{filename_prefix}_{name}_{global_step}_{score_name}={score}.{ext}"
+    _filename_pattern = "{filename_prefix}_{name}_{global_step}_{score_name}={score}.{ext}"
```

Comment:
`_filename_pattern` is not used, right ?

Reply:
yes you are right will fix this.